packages/angular/src/generators/library/lib/normalize-options.ts

```diff
@@ -34,7 +34,13 @@ export function normalizeOptions(
 
   const name = names(options.name).fileName;
   const projectDirectory = options.directory
-    ? `${names(options.directory).fileName}/${name}`
+    ? `${
+        names(
+          options.directory.endsWith('/')
+            ? options.directory.slice(0, -1)
+            : options.directory
+        ).fileName
+      }/${name}`
```

Comment:
Maybe we could simplify it and at the same time account for more cases where multiple `/` can be specified in the directory:
```suggestion
    ? `${names(options.directory).fileName}/${name}`.replace(/\/+/g, '/')
```

Reply:
Yep, much nicer!